Library/Pusher.h

```diff
@@ -11,3 +11,6 @@
 #import <Pusher/PTPusherEvent.h>
 #import <Pusher/PTPusherAPI.h>
 #import <Pusher/PTPusherEventDispatcher.h>
+#import <Pusher/PTPusher+Testing.h>
+#import <Pusher/PTPusherErrors.h>
+#import <Pusher/PTPusherMockConnection.h>
```

Comment:
These imports weren't previously necessary; why are they necessary now? Particularly the mock and testing stuff which should only be needed in tests.